src/components/messenger/list/conversation-item/conversation-item.tsx

```diff
@@ -1,19 +1,23 @@
 import * as React from 'react';
```

Comment:
We should probably stick to conventions and name this file `index.tsx`. Same for all the other ones.

Reply:
Okay. What would you name the `index.ts` file if renaming `conversation-item.tsx` to `index.tsx` ?

Reply:
I wouldn't have one. None of our other components have one.

Reply:
Okay no probs, the idea was an attempt to improve imports like 'import { menu } from './menu/menu`. I'll change those back. Thanks!

Reply:
Yeah, the issue is that doesn't follow any of the preexisting conventions and it's a pretty significant change to the file structure. If we want to propose new conventions those should come in separate PRs where the merits of the new proposal can be discussed outside the distraction of other changes.

Reply:
Nice yeah agree, doesn't make sense to have one component structured differently to the rest, understood that stirs up the existing conventions. Made those changes and checked everything is running okay :)

Reply:
:+1: 

Love the PR by the way. Thanks for cleaning up!